src/UpdatedPackagesCollector.js

```diff
@@ -13,7 +13,7 @@ export default class UpdatedPackagesCollector {
   constructor(command) {
     this.logger = command.logger;
     this.repository = command.repository;
-    this.packages = command.repository.packages;
+    this.packages = command.filteredPackages;
```

Comment:
I'm kind of against plain constructor tests, so I didn't provide any. If you would like me to I can though!